core/neat/mixins/_neat-column.scss

```diff
@@ -0,0 +1,39 @@
+/// Creates Neat a grid column of requested size.
+///
+/// @argument {number} $width
+///
+/// @argument {map} $grid
+///   The type of grid for this column. By default, the global
+///   `$neat-default-grid` will be used.
+///
+/// @example scss
+///   .element {
+///     @include neat-column(3);
+///   }
+///
+/// @example css
+///   .element {
+///     float: left;
+///     margin-left: 20px;
+///     width: calc();
+///   }
+
+@mixin neat-column($width: 1, $grid: $neat-default-grid) {
+  $_columns: map-get($grid, columns);
+  $_gutter: map-get($grid, gutter);
+
+  $_column-ratio: $width / $_columns;
+
+  @if $_gutter > 0 {
```

Comment:
Since there is branching logic in here, I am thinking about factoring the two width calculations into two `@function`s